content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,365 @@
+                Total time spent by all reduces in occupied slots (ms)=8865
+                Total time spent by all map tasks (ms)=5632
+                Total time spent by all reduce tasks (ms)=2955
+                Total vcore-seconds taken by all map tasks=5632
+                Total vcore-seconds taken by all reduce tasks=2955
+                Total megabyte-seconds taken by all map tasks=5767168
+                Total megabyte-seconds taken by all reduce tasks=9077760
+                DISK_MILLIS_MAPS=2816
+                DISK_MILLIS_REDUCES=3930
+        Map-Reduce Framework
+                Map input records=4
+                Map output records=4
+                Map output bytes=55
+                Map output materialized bytes=0
+                Input split bytes=181
+                Combine input records=0
+                Combine output records=0
+                Reduce input groups=2
+                Reduce shuffle bytes=65
+                Reduce input records=4
+                Reduce output records=2
+                Spilled Records=8
+                Shuffled Maps =1
+                Failed Shuffles=0
+                Merged Map outputs=2
+                GC time elapsed (ms)=209
+                CPU time spent (ms)=4600
+                Physical memory (bytes) snapshot=1240059904
+                Virtual memory (bytes) snapshot=9576202240
+                Total committed heap usage (bytes)=1421869056
+        HBase Counters
+                BYTES_IN_REMOTE_RESULTS=0
+                BYTES_IN_RESULTS=0
+                MILLIS_BETWEEN_NEXTS=0
+                NOT_SERVING_REGION_EXCEPTION=0
+                NUM_SCANNER_RESTARTS=0
+                NUM_SCAN_RESULTS_STALE=0
+                REGIONS_SCANNED=0
+                REMOTE_RPC_CALLS=0
+                REMOTE_RPC_RETRIES=0
+                ROWS_FILTERED=0
+                ROWS_SCANNED=0
+                RPC_CALLS=0
+                RPC_RETRIES=0
+        Shuffle Errors
+                IO_ERROR=0
+        File Input Format Counters
+                Bytes Read=0
+        File Output Format Counters
+                Bytes Written=0
+
+```
+
+
+
+**Important note**: If you encountered the issue that some of the HBase related Java packages are missing, you can simply copy the following packages from 📁<ins>/opt/mapr/hbase/hbase-{VERSION}/lib/</ins> to 📁<ins>/opt/mapr/hadoop/hadoop-{VERSION}/share/hadoop/common/</ins>.
+
+
+
+- hbase-client-\*.jar
+- hbase-server-\*.jar
+- hbase-protocol-\*.jar
+- hbase-hadoop2-compat-\*.jar
+- hbase-hadoop-compat-\*.jar
+- hbase-metrics-\*.jar
+- hbase-metrics-api-\*.jar
+- hbase-shaded-gson-\*.jar
+- hbase-shaded-htrace-\*.jar
+- metrics-core-\*.jar
+- hbase-common-\*.jar
+
+
+
+This is because the HBase related packages are considered as third-party libraries to the Hadoop system, refer to👉: [Install the third-party libraries on each node that runs the program][InstallTheThird-partyLibrariesOn].
```

Comment:
Rephrase first part to:
"You may run into this issue because the HBase related..."
End the sentence with "...to the Hadoop system."
Start the next sentence with "For more information, refer to:..."